package.json

```diff
@@ -20,11 +20,11 @@
     "test": "jest"
   },
   "dependencies": {
+    "@aws-sdk/client-dynamodb": "^3.609.0",
```

Comment:
Should we use ^ we don't want to mistakenly make major upgrades in future, that might break stuff

Reply:
Yeah _should_ just be compatible versions but I guess we can't be 100% sure with these AWS packages 😞 updated